cfn-resources/alert-configuration/cmd/resource/model.go

```diff
@@ -35,19 +21,19 @@ type Model struct {
 	Updated         *string               `json:",omitempty"`
 }
 
-// ApiKeyDefinition is autogenerated from the json schema
-type ApiKeyDefinition struct {
-	PrivateKey *string `json:",omitempty"`
-	PublicKey  *string `json:",omitempty"`
-}
-
 // Link is autogenerated from the json schema
 type Link struct {
 	ApiKeys *ApiKeyDefinition `json:",omitempty"`
 	Href    *string           `json:",omitempty"`
 	Rel     *string           `json:",omitempty"`
 }
 
+// ApiKeyDefinition is autogenerated from the json schema
+type ApiKeyDefinition struct {
+	PrivateKey *string `json:",omitempty"`
+	PublicKey  *string `json:",omitempty"`
+}
+
```

Comment:
this is not needed

Reply:
this APIkey is different. Links Property still uses this APIKeys Definition